charts/rancher-monitoring/v0.0.1/questions.yml

```diff
@@ -1 +1,2 @@
+rancher_min_version: 2.1.0-rc1
```

Comment:
These old monitoring charts have a max under `2.3.0-rc1` so I used min `2.1.0-rc1 because it's the oldest 2.x tag I found.